vars/pythonSetupPyPipeline.txt

```diff
@@ -42,9 +51,68 @@ Possibility to provide additional Docker run arguments.
 --network host
 ```
 
+### dockerDeploy
+
+Set to true if a Docker image with the Python module should be built and pushed to a Docker registry.
+
+### dockerRegistryUrl
+
+The Docker registry URL where the Docker image should be deployed to (eg. `https://registry.example.com`).
+
+### dockerRegistryCredentialsId
+
+Jenkins credentials id used to publish the Docker image to the Docker registry.
+
+### dockerRepo
+
+Name of the repository the Docker image will be deployed to.
+
+**Example**:
+
+For a Python module `foo` and `tomtom` as `dockerRepo` the image will be called `tomtom/foo`.
+
+The tag of the image is based on the version of the Python module version.
+
+### dockerDeployFile
+
+Dockerfile containing directives to build the image with the Python module.
+
+**Example**:
```

Comment:
Maybe it would make sense to link out to a project using this pipeline?
Alternatively, I would drop much of this, such as the "--extra-index-url" parameter and the LABEL to make it a more focused example.